RiverClan/settings.py

```diff
@@ -56,7 +56,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': ['templates'],
```

Comment:
I _think_ that should work, but to be safe, Amy did suggest this when someday else had an issue:
```suggestion
        "DIRS": [BASE_DIR / 'templates'],
```